components/coloring-listgroup.vue

```diff
@@ -61,8 +71,11 @@
 				for ( let index = 0; index < listGroup.children.length; index++ ) {
 					
 					// A. Decolor the list group item
-					listGroup.children[index].style.backgroundColor = this.defaultPalette.bColor;
-					listGroup.children[index].style.color = this.defaultPalette.fColor;
+					// listGroup.children[index].style.backgroundColor = this.defaultPalette.bColor;
```

Comment:
Yes, you're correct. And I had started working on that. But since the change would require some considerable working over and I wasn't sure if this would be our final change for category painting, I didn't want to go back and tackle that.

If you'd prefer, I can. But as it is right now since that information is stored as what determines the painted state of a row, it works as is regardless.

Reply:
oh, I just meant the comments here. Agree, no good point in reworking the default colors here. Sorry for not being clear. So: since we now set the item to opaque and the old code is commented out, let's remove the old (commented out) code and update the `II. ` enumeration in the comments

Reply:
Done. Though since I'm reworking the color, I'm thinking I may as well rework the cue for 'coloring' (really making opaque) the list group item. Hopefully it will just take a little bit of time today.

Reply:
@surchs Upon further inspection, the color cue makes sense. It is for the table that's being painted and not the list group with categories. I had mistakenly thought the stored `paintingData` was referencing the list group. So all good!